ETS2LA/modules/SDKController/main.py

```diff
@@ -92,7 +92,10 @@ def __init__(self):
         if system == "Windows":
             self._shm_buff = mmap.mmap(0, shm_size, self.MEM_NAME)
         elif system == "Linux":
-            self._shm_fd = open(self.SHM_FILE, "rb+")
+            try:
+                self._shm_fd = open(self.SHM_FILE, "rb+")
+            except:
+                raise RuntimeError(f"ETS2/ATS is not running (Currently game needs to be running for app to start (Blame tummy not me :3))") #Temporary "fix" to remind me that the game needs to be open, waiting for tummy to respond back on how to tell the app to stop using the sdk.
```

Comment:
Remove this part and it will be merged

Reply:
Should I just remove the extra text?

Not much I can do because I'm not sure how the inner of the app works.

Reply:
Just the blame tummy not me part

Reply:
Won't let me blame tummy x3.
Will do once I get some time in between classes.

Reply:
Sick